.github/workflows/clojure.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Get changed files
       id: changed-files
       run: |
-        if ${{ github.event_name == 'pull_request' }}; then
+        if ${{ github.event_name == 'pull_request' || github.event.before == '0000000000000000000000000000000000000000' }}; then
```

Comment:
Now we'll run the tests on the first push to a new branch.

Reply:
By run the tests on the first push to a new branch -- does this mean tests won't re-run if we push up a new commit?

Reply:
It will still run on pushes that aren't the first push. This just fixes a bug where you either had to open a PR or do another push to the branch to get the clojure tests to run.